aspnetcore/signalr/configuration.md

```diff
@@ -136,6 +137,21 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
 The following table describes options for configuring ASP.NET Core SignalR's advanced HTTP options:
 
+::: moniker range=">= aspnetcore-3.0"
+
+| Option | Default Value | Description |
+| ------ | ------------- | ----------- |
+| `ApplicationMaxBufferSize` | 32 KB | The maximum number of bytes received from the client that the server buffers before applying backpressure. Increasing this value allows the server to receive larger messages more quickly without applying backpressure, but can negatively impact memory consumption. |
```

Comment:
>negatively impact memory consumption

Is an interesting choice of words here lol.

Reply:
Yeah. "but can increase memory consumption" is probably a little clearer.